qt/qt_common/map_widget.cpp

```diff
@@ -420,10 +424,7 @@ void MapWidget::paintGL()
     int const samplerSizeLocation = m_program->uniformLocation("u_samplerSize");
     m_program->setUniformValue(samplerSizeLocation, samplerSize);
 
-    m_program->enableAttributeArray("a_position");
-    m_program->setAttributeBuffer("a_position", GL_FLOAT, 0, 4, 0);
-
-    funcs->glClearColor(0.0, 0.0, 0.0, 1.0);
+    funcs->glClearColor(0.5, 0.5, 0.5, 1.0);
```

Comment:
Is changing clear colors necessary?

Reply:
Color change is optional. I just got bored of black 😉